module/platformUserRoles/helper.js

```diff
@@ -185,4 +187,93 @@ module.exports = class platformUserRolesHelper {
 
     }
 
+    
+    static create(request, token) {
```

Comment:
@rakeshSgr - Please do mention parameter names that make sense and comments on top of a function that aptly describes the same.